### PR TITLE
Gladstone

### PR DESCRIPTION
**Pull request by Rat-Time**
Gladstone, back again, tested.
Please work.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/shotgun/gladstone.dm [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	mechanismvar = /obj/item/part/gun/mechanism/shotgun
+	barrelvars = list(/obj/item/part/gun/barrel/shotgun)
+
+/obj/item/gun/projectile/shotgun/pump/gladstone/sawn
+	name = "\"Gladstone\" stakeout shotgun"
+	desc = "A venerable shotgun that's been destroyed by some sort of heartless monster. Can hold up to 4+1 20mm shells in its tube magazine."
+	icon = 'icons/obj/guns/projectile/sawnoff/gladstone.dmi'
+	icon_state = "obrez"
+	item_state = "obrez"
+	gun_parts = list(/obj/item/part/gun/frame/gladstone = 1, /obj/item/stack/material/plastic = 1, /obj/item/part/gun/mechanism/shotgun = 1, /obj/item/stack/material/plasteel = 2)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: sawns shouldnt give back the frame iirc

Comment (Rat-Time) [reply code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: Was based on regulator/grizzly which both give frames when sawed off

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: Odd they really shouldnt, so you have a reason to use the repair repices

Comment (Rat-Time) [reply code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: Repairs.dm is commented out, you can't repair guns anymore

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: Fair

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/shotgun/gladstone.dm:42]: IMHO this is good. Frame + mechanism, the only tings that don't get fully manged(i guess we'll assume that stock=handle and barrel=barrel.)

**Pull request merged**